### PR TITLE
docs: updated fastify-session-authentication readme

### LINKED ISSUE #25: Provide example
The current example link targets the old library reference: https://github.com/fastify/example/tree/master/fastify-session-authentication

We need to adjust it.

### PR DESCRIPTION
**Pull request by dancastillo**
fixes fastify/example#25
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by mcollina — CHANGES_REQUESTED** [review]
Can you update the code too?

**Review by RafaelGSS — APPROVED** [review]

File: fastify-session-authentication/package.json [hunk 1]
```diff
@@ -10,14 +10,14 @@
...
+    "@fastify/formbody": "^7.0.1",
+    "@fastify/session": "^9.0.0",
+    "fastify": "^4.2.1"
   },
   "devDependencies": {
-    "fastify-cli": "^1.0.0",
-    "request": "^2.88.0",
-    "tap": "^14.1.7"
+    "fastify-cli": "^4.3.0",
+    "request": "^2.88.2",
```
Comment (mcollina) [inline fastify-session-authentication/package.json]: Maybe we want to drop request too (possibly in a follow-up PR)

**Review by mcollina — APPROVED** [review]
lgtm

File: fastify-session-authentication/package.json (hunk 1 above)
Comment (dancastillo) [reply fastify-session-authentication/package.json]: Good idea this PR is still open so will add to here

**Comment by Uzlopak** [comment]
Can you remove the yarn lock please?

**Pull request merged**